src/performance_profiles.jl

```diff
@@ -79,8 +79,16 @@ function performance_profile(T :: Array{Float64,2}, labels :: Vector{AbstractStr
   end
   Plots.xlims!(logscale ? 0.0 : 1.0, 1.1 * max_ratio)
   Plots.ylims!(0, 1.1)
-  Plots.xlabel!("Within this factor of the best" * (logscale ? " (log scale)" : ""))
-  Plots.ylabel!("Proportion of problems")
+  if :xlabel in keys(kwargs)
```

Comment:
It's a bit weird that the `kwargs` are already passed to `plot!()` on line 78. I feel like the x/y labels should be set on line 63 when the plot is initialized and if `:xlabel` or `:ylabel` are in `kwargs`, they should be `pop!()`d out of it.

Also maybe you can write `:xlabel in kwargs` instead of `:xlabel in keys(kwargs)` (I can't check now).

Reply:
I think you can do `haskey(kwargs, :xlabel)`.

Reply:
Apparently I can't use `pop!` with kwargs. 
If I push the changes I made on my local branch, will they show up on this pull request?

Reply:
Actually I have another question: is there a reason why the plot is created empty?
Why aren't the `title` and x/y lims parameters given to `plot` directly at line 63?

Reply:
That's what I'm saying we should do. First you need to convert `kwargs` to a `Dict`.